tools/contrib/junit_xml/__init__.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
```

Comment:
I'm also pretty skeptical of vendoring code, but I appreciate the desire to simplify the logic here, since it makes maintaining these test scripts more burdensome. I think, though, that significant disambiguation could be achieved just with better naming and organization: modularize the scripts so that environment set up and test logic are as orthogonal as possible, instead of the confusing nesting doll topology that we have now. The actual process graph doesn't seem to me like a significant impediment to understanding so much as which script is doing what.